Vagrantfile

```diff
@@ -39,7 +39,7 @@ Vagrant.configure("2") do |config|
   config.vm.define "salt-master" do |s_config|
      s_config.vm.provider "docker" do |docker, override|
        docker.build_dir = "dockerfiles"
-       docker.dockerfile = "Dockerfile.jammy"
+       docker.dockerfile = "Dockerfile.noble"
```

Comment:
If I recall there was an issue with upgrading the salt master related to one of the services. The looping thing?

Reply:
I thought that was for https://github.com/orgs/python/projects/47?query=sort%3Aupdated-desc+is%3Aopen&pane=issue&itemId=72813449 but I may be misremembering

Reply:
was able to apply in vagrant without issue
```
vagrant@salt-master:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 24.04 LTS
Release:        24.04
Codename:       noble
```

Reply:
https://github.com/saltstack/salt/pull/66330 is a bug in salt-master that is exposed when trying to provision cdn-logs. the bug happens when the salt-master tries to render the pillar data.

Reply:
to reproduce, I think you launch the 24.04 salt-master then try to provision cdn-logs (any base OS)